The Qt port in Tomoji's tree is totally outdate. The most recent version can be found in my branch. I would be happy if this one would be used as your starting point.

JensReimann opened the pull request.

JensReimann committed:
- update Qt port to latest VCL version

Commit:
Merge remote-tracking branch 'RTKLIB/rtklib_2.4.3' into rtklib_2.4.3

merge takasu's rtklib_2.4.3 b16

JensReimann committed:
update Qt port to takasu's rtklib_2.4.3 b16

- add include <clocale>
- some update project files

JensReimann committed:
suppress lots of warning

JensReimann committed:
suppress more warnings
remove unused function from septentrio.c

JensReimann committed:
Big Patch

- remove Qt4 support
- change may input from plain linedits to integer or double spin boxes
- use joint date & time input
- many cosmetic changes

JensReimann committed:
- small cleanup

JensReimann committed:
- add Visual C profile to project

JensReimann committed:
- improve build system

JensReimann committed:
- fix typo

JensReimann committed:
- remove no longer needed code

JensReimann committed:
bug fixes

- zoom in rtkplotqt
- date/time input widget time zone
- rtkpost_qt "abort" action

JensReimann committed:
update to latest version (2.4.3 R19)

JensReimann committed:
Updates

including:
- Improve scaling of window content
- fix milli seconds in QDateTime to gtime_t conersion
- fixes in srctblbrowser_qt
- add missing icons to resource files

JensReimann committed:
Merge remote-tracking branch 'Main/rtklib_2.4.3' into rtklib_2.4.3

JensReimann committed:
Update Qt code to latest changes in mainstream

JensReimann committed:
Merge remote-tracking branch 'Main/rtklib_2.4.3' into rtklib_2.4.3

JensReimann committed:
Update to latest version

JensReimann committed:
Merge remote-tracking branch 'Main/rtklib_2.4.3' into rtklib_2.4.3

JensReimann committed:
update to rtklib_2.4.3 b25

JensReimann committed:
Merge remote-tracking branch 'Main/rtklib_2.4.3' into rtklib_2.4.3

JensReimann committed:
update to rtklib_2.4.3 b26

JensReimann committed:
- small fix in strsvr_qt

JensReimann committed:
- Septentrio: delete removed signal frequencies and use rtklib constant
for glonass L3
- small bug fixes

JensReimann committed:
- fixed "NTRIP caster User-ID can't contain" (#5)
- update tab order in all dialogs
- fix mssing ico file on windows

JensReimann committed:
- fix stop button in strsvr_qt

JensReimann committed:
- fix display of RTCM messages in monitor dialog
- fix scroll down button in monitor dialog
- fix possible crash on shuwdown if monitor dialog is open

JensReimann committed:
add raw output to septentrio driver

JensReimann committed:
- fix scaling buttons in rtknavi_qt
- fix jumps at start of mouse dragging action

JensReimann committed:
fix periodic command sending causing issue #8

JensReimann committed:
- beautifying

JensReimann committed:
- fixed handling of additional start and stop commands to be send
- fixed so memory holes

JensReimann committed:
- fix memory holes

JensReimann committed:
- fix native suport of pathes using drag'n'drop
- fix alignment of widget in rtknavi_qt option dialog

JensReimann committed:
Fix to tcp option dialog

JensReimann committed:
- fix drag'n'drop issue in all remaining apps

JensReimann committed:
- UI improvement
- fix reading of minimum elevation angle from config file

JensReimann committed:
- fix GUI issu
	- checkbox was not updating control elements in cmdoptdlg
	- GLONASS issue and IODE/IODC were not displayed in monitor dialog of rtknavi_qt)
- minor cleanup in septentrio support (IODC is not transmitted at all
for Galileo and Galileo code is now correctly stored)

JensReimann committed:
Merge remote-tracking branch 'Main/rtklib_2.4.3' into rtklib_2.4.3

JensReimann committed:
update to b27

JensReimann committed:
- fix typos
- add some new .gitignore file
- add missing project files

tomojitakasu committed:
rtklib 2.4.3 b28

JensReimann committed:
Merge remote-tracking branch 'remotes/Main/rtklib_2.4.3' into rtklib_2.4.3

JensReimann committed:
- bump to b28

JensReimann committed:
- improve libiers compilation

JensReimann committed:
- small cleanup in septentrio receiver support

JensReimann committed:
-update .gitignore

JensReimann committed:
fix station file selection in rtkpost_qt option dialog

JensReimann committed:
- fix types in rtkconv_qt which causes input files 8+9 not working

tomojitakasu committed:
rtklib 2.4.3 b29

tomojitakasu committed:
rtklib 2.4.3 b29

tomojitakasu committed:
rtklib 2.4.3 b29

JensReimann committed:
Update to b29

Merge remote-tracking branch 'Main/rtklib_2.4.3' into rtklib_2.4.3

JensReimann committed:
-make use of IERS model an option (requires a fortran compiler)
-small fix in septrentrio driver

JensReimann committed:
- force c++11 needed for clang compiler

JensReimann committed:
Merge branch 'rtklib_2.4.3' of https://github.com/JensReimann/RTKLIB into rtklib_2.4.3

JensReimann committed:
fix startup tray option in rtknavi_qt

egorf commented:
Whoah, thanks for the PR. We'll dig through this and we'll write back. Would you be interested in working with us on advancing the Qt port?

JensReimann commented:
Hi,
I really like to see this Qt port being used. So, of cause I’d like to help.
 
At the moment I stuck as much as possible to the original code from Tomji to ease porting new features from his branch to the Qt port. I saw you’d like to improve code quality by doing larger modifications and refactoring. The only bigger modification I did (beside the port itself) was changing input widgets to validate the user input and some auto-completion for path inputs. 
The code is currently tested on linux (gcc & clang) and Windows 10 (msvc2015 & mingw). A Mac port should be easy but is not yet done.
 
Best regards,
Jens

tomojitakasu committed:
rtklib 2.4.3 b30

tomojitakasu committed:
rtklib 2.4.3 b30

tomojitakasu committed:
rtklib 2.4.3 b31

JensReimann committed:
Merge remote-tracking branch 'remotes/Main/rtklib_2.4.3' into rtklib_2.4.3

JensReimann committed:
adapted Qt version to patch level p31

JensReimann committed:
- fix save dialog in rtknavi-qt option dialog (was calling an open dialog instead)

JensReimann committed:
- limit fresh rate for graph in rtkplot_qt
- minor optimization in drawing code
- minor fixes

JensReimann committed:
- fix typo in rtknavi_qt/navimain.cpp causing rtkplot_qt call to fail

JensReimann committed:
- fix path to google maps file

JensReimann committed:
- don't write a new file when inserting API key for google maps view
- improve view alignment for GM and GE

JensReimann committed:
- load html code for google maps from header file instead of using a dedicated html file

tomojitakasu committed:
rtklib 2.4.3 b32

JensReimann committed:
Merge remote-tracking branch 'Main/rtklib_2.4.3' into rtklib_2.4.3

JensReimann committed:
- bump to version b32

holishing committed:
fix ws2_32 library name for case-sensitive

change Ws2_32 to ws2_32 for case-sensitive problem that makes link error in cross-compile environment for MinGW (Win32 excutable) by MXE toolchain in linux platform

MXE: https://mxe.cc/

JensReimann committed:
Merge pull request #21 from holishing/rtklib_2.4.3_b32p1

fix ws2_32 library name for case-sensitive

JensReimann committed:
- fix font selection dialog in rtkplot_qt

JensReimann committed:
Merge branch 'rtklib_2.4.3' of https://github.com/JensReimann/RTKLIB into rtklib_2.4.3

JensReimann committed:
- undo font selection fix for macOS
